### PR TITLE
[monorepo] setup nodewatch

### PR DESCRIPTION
**Pull request by AnthonyLaw**

### CODE REVIEW

**Comment by Jaguar0625** [comment]
This is fine to merge, but linter is failing. You can update the nodewatch code to comply with new linter rules in separate PR. As part of that, you should additionally add nodewatch badges to main project readme.

I would recommend squasing everything after feb1131 into single commit.

**Review by Jaguar0625 — APPROVED** [review]

File: nodewatch/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+defaultCiPipeline {
+	platform = ['ubuntu']
+	ciBuildDockerfile = 'python.Dockerfile'
+
+	packageId = 'nodewatch-puller'
+
+	codeCoverageTool = 'coverage'
+	minimumCodeCoverage = 95
+}
```
Comment (Wayonb) [inline nodewatch/Jenkinsfile:9]: Should remove the code coverage settings unless you plan to add tests.

```
	codeCoverageTool = 'coverage'
	minimumCodeCoverage = 95
```

File: nodewatch/scripts/ci/lint.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+../../../linters/python/lint.sh
```
Comment (Wayonb) [inline nodewatch/scripts/ci/lint.sh:1]: Should add ``#!/bin/bash``

**Review by Wayonb — CHANGES_REQUESTED** [review]
Looks good just a couple of small things.

File: nodewatch/scripts/ci/lint.sh (hunk 2 above)
Comment (Jaguar0625) [reply nodewatch/scripts/ci/lint.sh:1]: it is actually symbolic link, so should be fine

File: nodewatch/Jenkinsfile (hunk 1 above)
Comment (AnthonyLaw) [reply nodewatch/Jenkinsfile:9]: I will comment this line

File: nodewatch/scripts/ci/lint.sh (hunk 2 above)
Comment (AnthonyLaw) [reply nodewatch/scripts/ci/lint.sh:1]: This is a symlink for the linter file.
The linter file itself already has `#!/bin/bash`

**Review by Wayonb — APPROVED** [review]

**Review by Jaguar0625 — APPROVED** [review]

**Comment by Jaguar0625** [comment]
has anything changed since i last reviewed this? 😅

**Comment by AnthonyLaw** [comment]
I just rebase in my local, since I'm not able to do it here. 

the last change I made is the permission issue for Jenkins
https://github.com/symbol/product/pull/77/commits/5ed68d40862c9b941282ad4e190f8a03aa8de453

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**